tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_withdrawal_request.py

```diff
@@ -23,7 +24,7 @@ def test_basic_withdrawal_request(spec, state):
     state.slot += spec.config.SHARD_COMMITTEE_PERIOD * spec.SLOTS_PER_EPOCH
 
     current_epoch = spec.get_current_epoch(state)
-    validator_index = spec.get_active_validator_indices(state, current_epoch)[0]
+    validator_index = random.choice(spec.get_active_validator_indices(state, current_epoch))
```

Comment:
Hi @ensi321 ! Thank you for creating this PR. :)

To ensure test vectors are deterministic, all RNGs should be seeded per test case. e.g.,
```python
    rng = random.Random(1234)
    validator_index = rng.choice(spec.get_active_validator_indices(state, current_epoch))
```